### ISSUE TITLE
Entire ob shell command must be quoted

### ISSUE THREAD

**Issue by kmicklas** [obsidiansystems/obelisk#738]
Naively passing arguments to an `ob shell` command doesn't work:

```
> ob shell cabal new-test common
✔ Built on ./.obelisk/impl [command]
Invalid argument `new-test'

Usage: ob [-v|--verbose] COMMAND
  Manage Obelisk projects
```

However it does work if you quote the whole command:
```
> ob shell 'cabal new-test common'
✔ Built on ./.obelisk/impl [command]
Build profile: -w ghc-8.6.5 -O1
In order, the following will be built (use -v for more details):
 - common-0.1 (test:test) (first run)
Preprocessing test suite 'test' for common-0.1..
Building test suite 'test' for common-0.1..
Running 1 test suites...
Test suite test: RUNNING...
...
```

It would be nice to be able to pass arguments unquoted, either by requiring all `ob` options to appear before `shell` or with a `--` separator (like `cabal new-run`).

**Comment by 3noch** [comment]
I'm glad I'm not the only one who wants this.

**Comment by 3noch** [comment]
Fixed in #739
